packages/core/src/Stores/client-store.js

```diff
@@ -700,6 +701,10 @@ export default class ClientStore extends BaseStore {
         );
     }
 
+    get should_allow_poinc_authentication() {
```

Comment:
@yauheni-deriv Please add the type in `packages/stores/types.ts` 🙏🏻

Reply:
sure, thanks for this remark